src/gui/DatabaseWidget.h

```diff
@@ -153,6 +153,7 @@ class DatabaseWidget : public QStackedWidget
     void clearSearch();
     void requestGlobalAutoType(const QString& search);
     void requestSearch(const QString& search);
+    void updateLastDatabases(const QString& search);
```

Comment:
No need to introduce a new signal, just use databaseSaved.

Reply:
In this case I need to introduce `void updateLastDatabases(int index)` which will call the `void updateLastDatabases(const QString& filePath);` with the current file path.
```
void DatabaseTabWidget::updateLastDatabases(int index) {
    if (index == -1) {
        index = currentIndex();
    }

    auto filePath = databaseWidgetFromIndex(index)->database()->filePath();
    // FIXME: the filePath is empty here

    updateLastDatabases(filePath);
}
```
But this solution does not work.